### ISSUE TITLE
confusion conBase, refBase, and varBase

### ISSUE THREAD

**Issue by palomo11** [MrOlm/inStrain#3]
Hi, I'm checking the SNV output and it seems the refFreq is calculated based on the conBase instead of the refBase.

Here the output:

```
A | C | G | T | conBase | position | refBase | scaffold | varBase | cryptic | baseCoverage | varFreq | refFreq
4 | 0 | 2 | 0 | A | 675 | A | 10 | G | FALSE | 6 | 0.33 | 0.67
3 | 0 | 4 | 0 | G | 702 | G | 10 | A | FALSE | 7 | 0.43 | 0.57
0 | 2 | 0 | 3 | T | 717 | C | 10 | C | FALSE | 5 | 0.40 | 0.60
0 | 1 | 4 | 0 | G | 1741 | C | 10 | C | FALSE | 5 | 0.20 | 0.80
8 | 0 | 0 | 4 | A | 1769 | T | 10 | T | FALSE | 12 | 0.33 | 0.67
```

As you can see in the 3rd position, the refBase is C, therefore the refFreq should be 0.4, right? But instead, it says 0.6 (which is the conFreq).
I think it would be useful to get the three values, varFreq conFreq (the ones that are given now) and also the refFreq (so then it can be used to compare this value across different samples which share the position)

**Comment by MrOlm** [comment]
You are indeed correct and I've identified the bug in the code. Thank you for pointing this out to me, and the next version of the program will report all 3 freqs and refFreq correctly.

Thank you,
-Matt

**Comment by MrOlm** [comment]
Fixed in version 1.2.1 (and conFreq is now reported as well)
